src/UI/components/map/layers/drawerMap.tsx

```diff
@@ -36,8 +37,7 @@ export default function DrawerMap() {
     }),
     overflowX: 'hidden',
     margin: '0px',
-    marginTop: 18,
-    height: '80%',
+    height: 'calc(100vh - 230px)',
```

Comment:
Is 230px always going to be 230px?

Reply:
Probably not - at some point I need to push all of these into a setting in the theme and then we can adjust it in one go when we remove the alpha banner. Not a thing for now though.